includes/controls/popover-toggle.php

```diff
@@ -68,7 +68,7 @@ public function content_template() {
 					<i class="eicon-edit" aria-hidden="true"></i>
 					<span class="elementor-screen-only"><?php echo __( 'Edit', 'elementor' ); ?></span>
 				</label>
-				<input id="<?php echo $control_uid; ?>-default" type="radio" name="elementor-choose-{{ data.name }}-{{ data._cid }}" value="">
+				<input id="<?php echo $control_uid; ?>-default" class="elementor-control-popover-toggle-reset" type="radio" name="elementor-choose-{{ data.name }}-{{ data._cid }}" value="">
```

Comment:
Use `e-` prefix.

Reply:
This is an implementation of a predefined old selector